extensions/interactions/EndExploration/EndExploration.js

```diff
@@ -73,9 +74,15 @@ oppia.directive('oppiaInteractiveEndExploration', [
         if (!$scope.isInEditorPreviewMode) {
           authorRecommendationsPromise.then(function() {
             var explorationId = explorationContextService.getExplorationId();
+            var collectionSuffix = '';
+            if ($scope.collectionId) {
+              collectionSuffix = '?collection_id=' + $scope.collectionId;
```

Comment:
I feel there should be a UrlService.addQueryParameter(url, paramName, paramValue) method somewhere...